### DISCUSSION TITLE
What other names are for OSPOs?

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#16] [OSPO Discussions]
How do people call the open source initiatives within their organizations? I've seen folks calling them open-source centers, advisory councils, and more.

What other names are out there? :slightly_smiling_face:

**Comment by zhiqiang-yu** [comment]
OTC, Open Source Technology Center

**Reply by anajsana** [reply]
Interesting! Thanks for sharing :+1:

**Comment by caniszczyk** [comment]
I know IBM used "The Open Source Steering Committee (OSSC)"

http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.136.5657&rep=rep1&type=pdf

**Reply by c-rindi** [reply]
We call it the Open Source Software team.

**Comment by tisonkun** [comment]
Although we call it Community Development team, I'd prefer OSPO so that we share a standard for explaining and recruiting.

Also, behind OSPO we must define several focus (candidates) so that audiences get clear ideas about what we're doing.

For example, instead of community or open source barely, developer relationship is easier to understand which people can migrate knowledge from customer relationship or something.

**Reply by LawrenceHecht** [reply]
8-10 years ago, there were Formal Review Boards that looked at open source licensing and handled a lot of the related things. However, that is too narrow a definition now.

**Comment by Carl-Eric** [comment]
When I created the OSPO for Sony Ericsson (later renamed to Sony Mobile) in 2009 we called it "Open Source Software Operations", OSSO. There was a parallel entity for Software Security Operation, SSO. In addition we had a steering board called "Open Source Board" (which I - curiously enough - chaired as well) with cross-company representation (such as Legal, IPR, CTO, SW Dev and Business Development) that mostly was concerned on approving (larger) contributions.

**Reply by winterrocks** [reply]
Now I believe at Ericsson the office is called Open Source Office (OSO). Maybe someone from /// can confirm this?

**Reply by jsmanrique** [reply]
Cool to know. We are calling it OSO in the company I work for

**Reply by tsteenbe** [reply]
It's also called an Open Source Office in my company as we had already a programs office at the time so to avoid confusion we choose to use Open Source Office

**Comment by justinabrahms** [comment]
We call ours the "OSS Committee" or the "Open Source Software PMO".

**Comment by anajsana** [comment]
A new concept mentioned in the [TODO slack channel](https://thetodogroup.slack.com/archives/C0GBYUWCB/p1651873473378629) by @hyandell:

Acronym declare: FOSPO. Federated OSPO.  

When your OSPO's responsibilities are covered by multiple teams. I put forth that this is the goal for most OSPOs, for folk in Legal, Security, software-management, brand, and even dedicated-business unit OSPOs to all be a part of one company-spanning FOSPO

**Reply by anajsana** [reply]
Matt replied:

This certainly describes the experience we've seen at thousands of software companies- at least some "pure" OSPO functions are carried out by >1 department

**Comment by gvlx** [comment]
In Portugal you might find them as "Núcleo de Apoio ao Open Source" (support group).
